File: app/src/main/java/com/arman/queuetube/applications/BootstrapApplication.java
package com.arman.queuetube.applications;

import android.app.Application;

import com.beardedhen.androidbootstrap.TypefaceProvider;

public class BootstrapApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        TypefaceProvider.registerDefaultIconSets();
    }

}

File: app/src/main/java/com/arman/queuetube/applications/BootstrapApplication.kt
package com.arman.queuetube.applications

import android.app.Application

import com.beardedhen.androidbootstrap.TypefaceProvider

class BootstrapApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        TypefaceProvider.registerDefaultIconSets()
    }

}
